modules/statics/src/base.ts

```diff
@@ -319,7 +319,6 @@ export enum UnderlyingAsset {
   FET1 = 'fet1',
   FF1 = 'ff1',
   FFT = 'fft',
-  FIXED = 'fixed',
```

Comment:
probably we still want these underlying assets since 1 FIXED != 1 TERC20 != 1 GUSDT but I don't think it's a huge problem since we don't really care about pricing data for these. Up to you if you want to fix